README.md

```diff
@@ -76,22 +76,42 @@ Custom Slug Generation
 -------
 
 By default Mongoid Slug generates slugs with stringex. If this is not desired you can
-define your own slug generator like this:
-
-```ruby
-class Caption
-  include Mongoid::Document
-  include Mongoid::Slug
-
-  #create a block that takes the current object as an argument
-  #and returns the slug.
-  slug do |cur_object|
-    cur_object.slug_builder.to_url
-  end
-end
-```
+define your own slug generator.
+
+There are two ways to define slug generator
+
+1) Global definition:
```

Comment:
Clean this up. Ruby code should be as if it passes Rubocop. Maybe use `###` instead of 1) and 2).